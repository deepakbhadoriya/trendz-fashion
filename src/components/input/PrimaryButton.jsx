import React from 'react';

import styles from './PrimaryButton.module.scss';

const PrimaryButton = ({ title = 'button', className, onClick }) => {
  return (
    <button onClick={onClick} className={`${styles.button} ${className}`}>
      {title}
    </button>
  );
};

export default PrimaryButton;
