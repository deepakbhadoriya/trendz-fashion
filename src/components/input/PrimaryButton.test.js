import React from 'react';

import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import PrimaryButton from './PrimaryButton';

const { mount } = Enzyme;
Enzyme.configure({ adapter: new Adapter() });

describe('<PrimaryButton/>', () => {
  it('render correctly (snapshot)', () => {
    const tree = mount(<PrimaryButton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
