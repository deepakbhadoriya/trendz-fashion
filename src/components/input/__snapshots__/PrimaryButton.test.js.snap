// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrimaryButton/> render correctly (snapshot) 1`] = `
<PrimaryButton>
  <button
    className="button undefined"
  >
    button
  </button>
</PrimaryButton>
`;
